flaskr/app.py

```diff
@@ -96,4 +93,4 @@ def handle_standard_exception(ex):
 
 
 if __name__ == '__main__':
-    app.run()
+    bp.run()
```

Comment:
blueprints are not intended to run as it is. blueprints are modules of your API. blueprints are registered on Flask app level, and only Flask app is run.
https://flask.palletsprojects.com/en/1.1.x/blueprints/#registering-blueprints

Reply:
fixed